Source/Core/Duality/Utility/Coroutines/Conditions/WaitTime.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Duality.Utility.Pooling;
+
+namespace Duality
+{
+	public class WaitTime : WaitCondition<float>
+	{
+		public WaitTime(TimeSpan time) : base(
```

Comment:
Its simpler and more performant to just pass in a float here representing the amount of seconds

Reply:
Added a second constructor